easybuild/easyconfigs/l/libavif/libavif-1.0.4-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Denis Kristak (Inuits)
+# Update: Thomas Hoffmann (EMBL), Denis Kristak (Inuits), Pavel Tománek (Inuits)
+easyblock = 'CMakeMake'
+
+name = 'libavif'
+version = '1.0.4'
+
+homepage = 'https://github.com/AOMediaCodec/libavif'
+description = """This library aims to be a friendly, portable C implementation of the AV1 Image File Format,
+as described here: https://aomediacodec.github.io/av1-avif/
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+source_urls = ['https://github.com/AOMediaCodec/libavif/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['dc56708c83a4b934a8af2b78f67f866ba2fb568605c7cf94312acf51ee57d146']
+
+builddependencies = [
+    ('CMake', '3.26.3'),
+    ('binutils', '2.40'),
+]
+
+dependencies = [
+    ('NASM', '2.16.01'),
+    ('Meson', '1.1.1'),
+    ('Ninja', '1.11.1'),
+    ('Rust', '1.75.0'),
```

Comment:
@pavelToman Is `Rust` 1.70.0 sufficiently recent? If so, let's stick to that.

Alternatively (and maybe better), `Rust` is probably a build-only dependency for this, and same goes for Meson + Ninja I think...

Reply:
I moved all the deps to builddeps - seems the `NASM` could be there either - it is `General-purpose x86 assembler` and in some easyconfigs is in builddeps.